Spring.FluentContext.sln

```diff
@@ -1,69 +1,121 @@
-		{7D461C3F-F00E-4F67-A083-7184ED2E8128}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{7D461C3F-F00E-4F67-A083-7184ED2E8128}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{7D461C3F-F00E-4F67-A083-7184ED2E8128}.Release|Any CPU.Build.0 = Release|Any CPU
-		{7D461C3F-F00E-4F67-A083-7184ED2E8128}.Release|Mixed Platforms.ActiveCfg = Release|Any CPU
-		{7D461C3F-F00E-4F67-A083-7184ED2E8128}.Release|Mixed Platforms.Build.0 = Release|Any CPU
-		{7D461C3F-F00E-4F67-A083-7184ED2E8128}.Release|x86.ActiveCfg = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(SolutionProperties) = preSolution
-		HideSolutionNode = FALSE
-	EndGlobalSection
-	GlobalSection(ExtensibilityGlobals) = postSolution
-		SolutionGuid = {B7744C9E-9E0F-478B-8731-22B5B1609E34}
-	EndGlobalSection
-	GlobalSection(MonoDevelopProperties) = preSolution
-		StartupItem = Spring.FluentContext.Examples\Spring.FluentContext.Examples.csproj
-	EndGlobalSection
-EndGlobal
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.28307.168
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Spring.FluentContext", "Spring.FluentContext\Spring.FluentContext.csproj", "{166FE8E4-E3E6-4A01-9D73-890E6E0F31A6}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Spring.FluentContext.UnitTests", "Spring.FluentContext.UnitTests\Spring.FluentContext.UnitTests.csproj", "{4397BF4A-8180-4F06-91BC-7AC387E53C45}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Spring.FluentContext.Examples", "Spring.FluentContext.Examples\Spring.FluentContext.Examples.csproj", "{7D461C3F-F00E-4F67-A083-7184ED2E8128}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{AF807B3F-7ABB-4503-AB5C-7DD532960395}"
+	ProjectSection(SolutionItems) = preProject
+		Changelog.txt = Changelog.txt
+		LICENSE.txt = LICENSE.txt
+		README.md = README.md
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "net35", "net35", "{EB78084A-C54B-4ED4-BF8C-D197BCC2F83F}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Spring.FluentContext", "net35\Spring.FluentContext\Spring.FluentContext.csproj", "{D807FE64-2112-4977-9BFE-2B0D879C5CAB}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Spring.FluentContext.UnitTests", "net35\Spring.FluentContext.UnitTests\Spring.FluentContext.UnitTests.csproj", "{F6B4F58A-676D-4089-9B62-A14B29BD459A}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Spring.FluentContext.Examples", "net35\Spring.FluentContext.Examples\Spring.FluentContext.Examples.csproj", "{08F8ED71-D9B6-4A1C-9AA7-3183E4C82452}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Spring.FluentContext.Web", "Spring.FluentContext.Web\Spring.FluentContext.Web.csproj", "{9FC6E664-D984-4D1F-B637-79709F958CA2}"
+EndProject
```

Comment:
The build is failing due to this sln file content. Please fix it: https://ci.appveyor.com/project/Suremaker/spring-fluentcontext/builds/22379526